yt_dlp/extractor/veoh.py

```diff
@@ -123,3 +130,91 @@ def _real_extract(self, url):
             'categories': categories,
             'tags': tags.split(', ') if tags else None,
         }
+
+
+class VeohUserIE(VeohIE):
+    _VALID_URL = r'https?://(?:www\.)?veoh\.com/users/(?P<user>([\da-zA-Z_\-]+))'
+    IE_NAME = 'veoh:user'
+
+    _TESTS = [
+        {
+            'url': 'https://www.veoh.com/users/valentinazoe',
+            'info_dict': {
+                'id': 'valentinazoe',
+                'title': 'valentinazoe (Uploads)'
+            },
+            'playlist_mincount': 75
+        },
+        {
+            'url': 'https://www.veoh.com/users/PiensaLibre',
+            'info_dict': {
+                'id': 'PiensaLibre',
+                'title': 'PiensaLibre (Uploads)'
+            },
+            'playlist_mincount': 2
+        }]
+
+    _USERSINFO_ENDPOINT = 'https://www.veoh.com/users/published/videos'
+    _VIDEO_BASEURL = 'https://www.veoh.com/watch/'
+
+    def _get_authtoken(self, page, uploader=None):
+        webpage = self._download_webpage(page, uploader)
+        return self._search_regex(
+            r'csrfToken: "(?P<token>[0-9a-zA-Z]{40})"', webpage,
```

Comment:
I'd relax the regex a little bit

```suggestion
            r'csrfToken:\s*(["\'])(?P<token>[0-9a-zA-Z]{40})\1', webpage,
```
is the csrf token always 40 characters long?

Reply:
Hi Colet! thank you for your suggestions, I implemented them.
> is the csrf token always 40 characters long?

Yes, it is my understanding that they are always 40 characters long.